### PR TITLE
update bind to latest stable series release

### PR DESCRIPTION
**Pull request by satmandu**
Works properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=bind CREW_TESTING=1 crew update
```

### CODE REVIEW

File: manifest/x86_64/b/bind.filelist [hunk 1]
```diff
@@ -259,60 +251,29 @@
...
-/usr/local/share/man/man1/nsupdate.1.zst
-/usr/local/share/man/man5/named.conf.5.zst
-/usr/local/share/man/man5/rndc.conf.5.zst
-/usr/local/share/man/man8/ddns-confgen.8.zst
-/usr/local/share/man/man8/filter-a.8.zst
-/usr/local/share/man/man8/filter-aaaa.8.zst
-/usr/local/share/man/man8/named.8.zst
-/usr/local/share/man/man8/rndc.8.zst
-/usr/local/share/man/man8/rndc-confgen.8.zst
-/usr/local/share/man/man8/tsig-keygen.8.zst
```
Comment (uberhacker) [inline manifest/x86_64/b/bind.filelist:318]: What happened to the manpages?

Comment (satmandu) [reply manifest/x86_64/b/bind.filelist:318]: I just changed the version numbers and built...
Are those extra man pages from the newer version that got removed? Or all of them?

Comment (uberhacker) [reply manifest/x86_64/b/bind.filelist:318]: I have no idea.  Might need to include a switch option for the manpages in the stable release.

File: packages/bind.rb [hunk 2]
```diff
@@ -3,23 +3,23 @@
 class Bind < Package
   description 'BIND is open source software that enables you to publish your Domain Name System (DNS) information on the Internet, and to resolve DNS queries for your users.'
   homepage 'https://www.isc.org/downloads/bind/'
-  version '9.19.9'
+  version '9.18.16'
   license 'Apache-2.0, BSD, BSD-2, GPL-2, HPND, ISC and MPL-2.0'
-  compatibility 'all'
+  compatibility 'x86_64'
```
Comment (uberhacker) [inline packages/bind.rb:8]: Why was this changed?

Comment (satmandu) [reply packages/bind.rb:8]: Oops. That's a bug from me not fixing up a file...

File: manifest/x86_64/b/bind.filelist (hunk 1 above)
Comment (satmandu) [reply manifest/x86_64/b/bind.filelist]: Fixed.

File: packages/bind.rb (hunk 2 above)
Comment (satmandu) [reply packages/bind.rb]: Fixed.

**Review by uberhacker — APPROVED** [review]
